OWNERS.md

```diff
@@ -39,14 +39,15 @@ Being part of Technical Steering Committee (TSC) [@StackStorm/maintainers](https
   - Systems, ST2 Exchange. [Case Study](https://stackstorm.com/case-study-bitovi/).
 * Nick Maludy ([@nmaludy](https://github.com/nmaludy)) <<nmaludy@gmail.com>>
   - Community, Core, Systems, StackStorm Exchange, Puppet deployment. [Case Study](https://stackstorm.com/case-study-dmm/).
+* Carlos ([@nzlosh](https://github.com/nzlosh))
```

Comment:
We'll need a working email here to follow the same format for TSC members.

Reply:
We can use `nzlosh@yahoo.com`.